src/app/modules/library/library-project/library-project.component.ts

```diff
@@ -55,4 +65,9 @@ export class LibraryProjectComponent implements OnInit {
       panelClass: 'dialog-md'
     });
   }
+
+  protected projectSelected(event: any): void {
```

Comment:
Make function name more active?
```suggestion
  protected selectProject(event: any): void {
```